src/OpenAL/OpenToolkit.OpenAL/AL/AL.cs

```diff
@@ -974,11 +973,21 @@ public static void BufferData<TBuffer>(int bid, ALFormat format, TBuffer[] buffe
         public static void BufferData<TBuffer>(int bid, ALFormat format, Span<TBuffer> buffer, int freq)
             where TBuffer : unmanaged
         {
+#if NETCOREAPP3_1
```

Comment:
Just a question, can we replace `NETCOREAPP3_1` for `NETCOREAPP`

Reply:
The feature used (`MemoryMarshal`) is from .net core 2.1 so it wouldn't work. It does exist in .net standard 2.1 so we could use `#if NETSTANDARD2_1` instead.